test/plain-list/AutosuggestApp.test.js

```diff
@@ -229,6 +479,25 @@ describe('Plain list Autosuggest', () => {
     });
   });
 
+  describe('when pressing Enter and selectFirstSuggestion is true', () => {
+    before(() => {
+      setSelectFirstSuggestion(true);
+    });
+
+    after(() => {
+      setSelectFirstSuggestion(false);
+    });
+
+    beforeEach(() => {
+      focusAndSetInputValue('p');
+    });
+
+    it('should hide suggestions since the first suggestion is focused', () => {
+      clickEnter();
+      expectSuggestions([]);
+    });
```

Comment:
This did remind me that we needed the same test for mouse hover events, which I added in https://github.com/moroshko/react-autosuggest/pull/165/commits/a3dba9f3e5e3d2e812fe9bb84abe997b135ddfc5